scripts/Muon/GUI/Common/plot_widget/plotting_canvas/plotting_canvas_view.py

```diff
@@ -148,13 +157,29 @@ def add_workspaces_to_plot(self, workspace_plot_info_list: List[WorkspacePlotInf
             axis_number = self._make_plot(workspace_plot_info)
             if axis_number < 0:
                 continue
+            self._set_text_tick_labels(axis_number)
             if self._settings.is_condensed:
                 self.hide_axis(axis_number, nrows, ncols)
         #remove labels from empty plots
         if self._settings.is_condensed:
             for axis_number in range(int(self._number_of_axes), int(nrows*ncols)):
                 self.hide_axis(axis_number, nrows, ncols)
 
+    def _wrap_labels(self, labels: list) -> list:
+        """Wraps a list of labels so that every line is at most LABEL_WRAP_WIDTH characters long."""
```

Comment:
Where is LABEL_WRAP_WIDTH defined? It also does not appear to be used in the function.

Reply:
I replaced it with this self._settings.wrap_width

Reply:
```suggestion
        """Wraps a list of labels so that every line is at most self._settings.wrap_width characters long."""
```